<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>536</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>101</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="datelabel">
       <property name="text">
        <string>系统时间</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="timelabel">
       <property name="text">
        <string>系统日期</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>cpu</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_cpu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>内存</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_mem">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>进程信息</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_pro">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab4">
     <attribute name="title">
      <string>详细信息</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_com">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="datelabel_2">
       <property name="text">
        <string>CPU使用率</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <widget class="QLabel" name="cpu_usage">
       <property name="text">
        <string>cpu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="datelabel_3">
       <property name="text">
        <string> 内存使用率</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="1" column="0">
      <widget class="QLabel" name="mem_usage">
       <property name="text">
        <string>memory</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menulinux">
    <property name="title">
     <string>linux实验</string>
    </property>
   </widget>
   <addaction name="menulinux"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
